src/tabs/worldMap/WorldMapTabView.tsx

```diff
@@ -28,24 +28,28 @@ export class WorldMapTabView extends TabView<WorldMapTabState> {
           <RX.View style={Styles.flex.row}>
             <RX.Text>X:</RX.Text>
             <RX.TextInput
+              defaultValue={"0"}
               style={[Styles.whiteBox, Styles.flex.fill, Styles.smallTextInput]}
               onChangeText={(x) => this.center.x = Number(x)}/>
           </RX.View>
           {this.renderYCoordinateInput(this.surface)}
           <RX.View style={Styles.flex.row}>
             <RX.Text>Z: </RX.Text>
             <RX.TextInput
+              defaultValue={"0"}
               style={[Styles.whiteBox, Styles.flex.fill, Styles.smallTextInput]}
               onChangeText={(z) => this.center.z = Number(z)}/>
           </RX.View>
         </RX.View>
         <RX.View style={Styles.flex.row}>
           <RX.Text>Width:</RX.Text>
           <RX.TextInput
+            defaultValue={"10"}
```

Comment:
I would use larger default values, IMHO a 10x10 map doesn't really give an idea of how the world map looks like in-game, maybe I'd put something like 25x25? Do you agree?